src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -0,0 +1,144 @@
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'MonoAndroid70'">
+    <TargetFrameworkIdentifier>MonoAndroid</TargetFrameworkIdentifier>
+    <TargetFrameworkVersion>v7.0</TargetFrameworkVersion>
+    <AndroidUseLatestPlatformSdk>False</AndroidUseLatestPlatformSdk>
+    <DebugType>full</DebugType>
+    <DefineConstants>$(DefineConstants);MONOANDROID7_0</DefineConstants>
+    <AndroidResgenFile>Resources\Resource.Designer.cs</AndroidResgenFile>
+    <LanguageTargets>$(MSBuildProgramFiles32)\MSBuild\Xamarin\Android\Xamarin.Android.CSharp.targets</LanguageTargets>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net45'">
+    <DefineConstants>$(DefineConstants);NET4_5</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="**\*.cs" Exclude="Platforms\**\*.*;Resources\**\*.cs" />
+    <!-- This is here so that the conditionally included files still show up in VS -->
+    <None Include="**\*.cs;**\*.xml;**\*.axml" Exclude="obj\**\*.*;bin\**\*.*" />
+    <EmbeddedResource Include="Properties\Microsoft.Identity.Client.rd.xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="NETStandard.Library" Condition="'$(TargetFramework)' != 'net45'" Version="1.6.1" />
+    <PackageReference Include="System.Runtime.Serialization.Primitives" Condition="'$(TargetFramework)' != 'net45'" Version="4.3.0" />
+    <PackageReference Include="System.Runtime.Serialization.Json" Condition="'$(TargetFramework)' != 'net45'" Version="4.3.0" />
+    <PackageReference Include="System.Security.Cryptography.X509Certificates" Condition="'$(TargetFramework)' == 'netstandard1.3' or '$(TargetFramework)' == 'MonoAndroid70' or '$(TargetFramework)' == 'Xamarin.iOS10' or '$(TargetFramework)' == 'uap10.0' " Version="4.3.0" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.IdentityModel" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Xml" />
+    <Compile Include="Platforms\net45\*.cs" />
+    <Compile Include="Platforms\Common\MsalEventSource.cs" />
+    <Compile Include="Platforms\Common\PublicClientApplicationExtensions.cs" />
+    <Compile Include="Platforms\Common\ClientAssertionCertificate.cs" />
+    <Compile Include="Platforms\Common\Logger.cs" />
+    <Compile Include="Platforms\Common\MsalOption.cs" />    
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'wpa81' or '$(TargetFramework)' == 'win81' or '$(TargetFramework)' == 'uap10.0' ">
```

Comment:
we don't plan on supporting wpa81. I think should not have defined in the csproj.